src/battle_script_commands.c

```diff
@@ -3148,7 +3148,12 @@ void SetMoveEffect(bool32 primary, u32 certain)
                         BattleScriptPush(gBattlescriptCurrInstr + 1);
                         gBattlescriptCurrInstr = BattleScript_MoveEffectFeint;
                     }
-                }
+				}
+				if (gCurrentMove == MOVE_HYPERSPACE_FURY)
+				{
+					BattleScriptPush(gBattlescriptCurrInstr + 1);
+					gBattlescriptCurrInstr = BattleScript_DefDown;
+				}
```

Comment:
Spaces instead of tabs.

Also, not a fan of hardcoding the effect to Hyperspace Fury specifically like this, personally.

Reply:
Me neither, .argument wasn't working properly though.  It would say something about there being too many characters.  Will change to spaces.